### PR TITLE
loosen activeadmin restriction to allow 2.0

### PR DESCRIPTION
**Pull request by iarie**
https://github.com/activeadmin/activeadmin/blob/master/CHANGELOG.md

Activeadmin 2.0 do not сontain any major changes except ruby 2.3 support drop, this gem should be fine

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
Hi @iarie, sorry if the changelog is confusing.

The ruby 2.3 support drop has not yet been released. The list of changes in the 2.0 release can be found mainly under the 2.0.0.rc1 entry. The fact that the 2.0.0 entry says "No changes" means that there were no user visible changes between 2.0.0.rc2 and 2.0.0 final.

**Comment by workgena** [comment]
I'll take a look at this PR tomorrow :sun_with_face:

File: active_admin_datetimepicker.gemspec [hunk 1]
```diff
@@ -19,6 +19,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activeadmin", "~> 1.1"
+  spec.add_dependency "activeadmin", ">= 1.1", "< 3.x"
```
Comment (deivid-rodriguez) [inline active_admin_datetimepicker.gemspec]: If the intention is to not allow future ActiveAdmin 3.0.0 prereleases, then the correct way to do that is:

`spec.add_dependency "activeadmin", ">= 1.1", "< 3.a"`

**Comment by iarie** [comment]
@deivid-rodriguez hi, yes but there are no major changes  in rc1 section as well, so I thought this gem should work fine on AA 2.0

File: active_admin_datetimepicker.gemspec (hunk 1 above)
Comment (iarie) [reply active_admin_datetimepicker.gemspec]: Thank you, my intention was exactly like that

**Comment by deivid-rodriguez** [comment]
> @deivid-rodriguez hi, yes but there are no major changes in rc1 section as well, so I thought this gem should work fine on AA 2.0

Ah, ok. Since you said that the main breaking change was dropping support for ruby 2.3, I thought you had misunderstood the changelog :+1:. It sounds like the only thing needed to get CI green would be to drop Rails 4.2 support in this plugin just like AA did.

**Comment by workgena** [comment]
Firstly, @iarie thanx for contribution.

I've found what was the problem with ActiveAdmin 2.0 and fixed in followed commit.

Then I decided to remove 2.3 ruby support due to:

[Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/)

Ruby 2.3

status: eol
release date: 2015-12-25
EOL date: 2019-03-31

---

[Rails Supported versions](https://rubyonrails.org/security/)

For major security issues, all releases in the current major series, and the last release in the additional major series will receive patches and new versions. This is currently 5.2.x, 5.1.x, 5.0.x and 4.2.x.

**Comment by workgena** [comment]
I'll do version bump after merging this PR. Before that I want to check it manually.

**Comment by workgena** [comment]
Checked it manually. Works fine.

@iarie @deivid-rodriguez If you do not have any other comments on this PR, I'll merge it.

**Comment by iarie** [comment]
Thank you, @workgena!

**Review by workgena — APPROVED** [review]

**Pull request merged**